.github/workflows/infracost.yaml

```
@@ -0,0 +1,60 @@
+name: Infracost exibe estimativa de custo da infraestrutura
+
+on:
+  workflow_call:
+    inputs:
+      OS_VERSION:
+        description: "Versão do sistema operacional"
+        required: false
+        default: "ubuntu-latest"
+        type: string
+      TF_DIRECTORY:
+        description: "Diretório do projeto terraform"
+        required: false
+        default: .
+        type: string
+    secrets:
+      INFRACOST_API_KEY:
+        description: "APIKEY do Infracost"
+        required: true
+jobs:
+  infracost:
+    name: Infracost workflow
+    runs-on: ${{ inputs.OS_VERSION }}
+    env:
+       INFRACOST_ENABLE_CLOUD: true
+    steps:
+      - name: Configura Infracost
+        uses: infracost/actions/setup@v2
+        with:
+          api-key: ${{ secrets.INFRACOST_API_KEY }}
+
+      - name: Checkout base branch
+        uses: actions/checkout@v2
+        with:
+          ref: '${{ github.event.pull_request.base.ref }}'
+
+      - name: Gerando arquivo Infracost custo estimado
```

```suggestion
      - name: Gerando estimativa de custo atual
```

Eu não tinha entendido muito bem o que esse passo faz. Acho que ajuda usar um nome mais descritivo.